Sources/BraintreeVenmo/BTVenmoError.swift

```diff
@@ -57,6 +57,9 @@ enum BTVenmoError: Error, CustomNSError, LocalizedError {
 
     /// 9. Enriched Customer Data is disabled
     case enrichedCustomerDataDisabled
+    
+    /// 10.  The Venmo flow was canceled by the user
+    case canceled
```

Comment:
q: is this a breaking change?

Reply:
Nope, since we are only adding an error code & not changing existing ones we didn't think this was breaking